modules/reference/pages/rpk/rpk-topic/rpk-topic-consume.adoc

```diff
@@ -7,296 +7,203 @@ according to `--format`, and prints them to `STDOUT`. The output formatter
-    @t1:t2    consume from timestamp t1 until timestamp t2
-----
+[cols=",",]
+|===
+|Offset |Description
+
+|`start` |Consume from the beginning
+|`end` |Consume from the end
+|`:end` |Consume until the current end
+|`+oo` |Consume oo after the current start offset
+|`-oo` |Consume oo before the current end offset
+|`oo` |Consume after an exact offset
+|`oo:` |Alias for oo
+|`:oo` |Consume until an exact offset
+|`o1:o2` |Consume from exact offset o1 until exact offset o2
+|`@t` |Consume starting from a given timestamp
+|`@t:` |alias for @t
+|`@:t` |Consume until a given timestamp
+|`@t1:t2` |Consume from timestamp t1 until timestamp t2
+|===
 
-There are a few options for timestamps, with each option being evaluated
-until one succeeds:
+Each timestamp option is evaluated until one succeeds.
 
-[.no-copy]
-----
-    13 digits             parsed as a unix millisecond
-    9 digits              parsed as a unix second
-    YYYY-MM-DD            parsed as a day, UTC
-    YYYY-MM-DDTHH:MM:SSZ  parsed as RFC3339, UTC; fractional seconds optional (.MMM)
-    end                   for t2 in @t1:t2, the current end of the partition
-    -dur                  a negative duration from now or from a timestamp
-    dur                   a positive duration from now or from a timestamp
-----
+[cols=",",]
+|===
+|Timestamp |Description
 
-Durations can be relative to the current time or relative to a timestamp.
-If a duration is used for t1, that duration is relative to now.
-If a duration is used for t2, if t1 is a timestamp, then t2 is relative to t1.
-If a duration is used for t2, if t1 is a duration, then t2 is relative to now.
+|13 digits |Parsed as a unix millisecond
 
-Durations are parsed simply:
+|9 digits |Parsed as a unix second
 
-[.no-copy]
-----
-    3ms    three milliseconds
-    10s    ten seconds
-    9m     nine minutes
-    1h     one hour
-    1m3ms  one minute and three milliseconds
-----
+|YYYY-MM-DD |Parsed as a day, UTC
 
-For example,
+|YYYY-MM-DDTHH:MM:SSZ |Parsed as RFC3339, UTC; fractional seconds
+optional (.MMM)
 
-[.no-copy]
-----
-    -o @2022-02-14:1h   consume 1h of time on Valentine's Day 2022
-    -o @-48h:-24h       consume from 2 days ago to 1 day ago
-    -o @-1m:end         consume from 1m ago until now
-    -o @:-1hr           consume from the start until an hour ago
-----
+|-dur |Duration; from now (as t1) or from t1 (as t2)
+
+|dur |For t2 in @t1:t2, relative duration from t1
+
+|end |For t2 in @t1:t2, the current end of the partition
+|===
+
+Durations are parsed simply:
+
+```
+3ms    three milliseconds
+10s    ten seconds
+9m     nine minutes
+1h     one hour
+1m3ms  one minute and three milliseconds
+```
+
+For example:
+
+```
+-o @2022-02-14:1h   consume 1h of time on Valentine's Day 2022
+-o @-48h:-24h       consume from 2 days ago to 1 day ago
+-o @-1m:end         consume from 1m ago until now
+-o @:-1hr           consume from the start until an hour ago
+```
+
+=== Misc
```

Comment:
@Deflaimun I don't think we should use **Misc** as a heading. What do you think about Notes? If you think it's important to keep, then I suggest using Miscellaneous

Reply:
You're right. This info better fits in the description